Resources/public/js/ezrecommendation.js

```diff
@@ -0,0 +1,77 @@
+// register HandleBars helpers when library is loaded
```

Comment:
I thought it's better if user can embed only one file instead of 3 files.
I think the main problem with this task is that no detailed specification (user requirements, expectations) were prepared.